demo/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "build": "webpack",
-    "serve": "npm i && webpack-dev-server --open",
+    "serve": "webpack-dev-server --open",
     "start": "node scripts/start.js npm run serve"
```

Comment:
```suggestion
    "serve": "npm i && webpack-dev-server --open",
    "start": "npm run build && npm run serve"
```

Reply:
I updated the demo/package.json to the following based on your suggestions:
```js
    "scripts": {
      "build": "NODE_OPTIONS=--openssl-legacy-provider webpack",
      "serve": "npm i && webpack-dev-server --open",
      "start": "npm run build && npm run serve"
    },
```
Then I deleted my node_modules and package-lock.json, and set my node version to v19.4.0. Finally, I ran `npm start` and received an error: `sh: webpack: command not found`. This means that `npm install` needs to be run before running `npm start`, which is what Oana had to do to fix her issue. That's the reason I ended up removing it from the serve command. Let me know if I'm missing something!

Reply:
Gotcha, thanks for experimenting with it - we can have the user run `npm install` separately from `npm build` and `npm serve` then!